commands/create_cluster.go

```diff
@@ -470,15 +470,17 @@ func createAddClusterBody(d clusterDefinition) *models.V4AddClusterRequest {
 		Max: d.Scaling.Max,
 	}
 
-	for _, dWorker := range d.Workers {
+	if len(d.Workers) >= 1 {
```

Comment:
In which cases is it greater than 1?

Reply:
There is still some old behaviour that creates more than one worker with the `--num-workers` flag. I'll sort it out. Thanks for reading carefully!